pycytominer/cyto_utils/cells.py

```diff
@@ -667,11 +670,14 @@ def merge_single_cells(
             Whether or not to normalize the single cell data.
         normalize_args : dict, optional
             Additional arguments passed as input to pycytominer.normalize().
+        platemap: str or pd.DataFrame, default None
+            optional platemap filepath str or pd.DataFrame to be used with results via annotate
 
         Returns
         -------
-        pandas.core.frame.DataFrame
-            Either a dataframe (if output_file="none") or will write to file.
+        pandas.core.frame.DataFrame or str
+            if output_file="none") returns a Pandas dataframe
```

Comment:
```suggestion
            if output_file="none" returns a Pandas dataframe
```

Reply:
Thank you! I'll add this to a batch of changes.